cartography/intel/aws/iam.py

```diff
@@ -622,8 +629,11 @@ def sync_user_access_keys(neo4j_session, boto3_session, current_aws_account_id,
     query = "MATCH (user:AWSUser)<-[:RESOURCE]-(AWSAccount{id: {AWS_ACCOUNT_ID}}) return user.name as name"
     result = neo4j_session.run(query, AWS_ACCOUNT_ID=current_aws_account_id)
     usernames = [r['name'] for r in result]
-    account_access_key = {name: get_account_access_key_data(boto3_session, name) for name in usernames}
-    load_user_access_keys(neo4j_session, account_access_key, aws_update_tag)
+    for name in usernames:
+        access_keys = get_account_access_key_data(boto3_session, name)
+        if access_keys:
+            account_access_keys = {name: access_keys}
+            load_user_access_keys(neo4j_session, account_access_keys, aws_update_tag)
```

Comment:
Short term fix to handle errors causing crashes. 

Long term: need to look at performance of single vs bulk changes to neo4j that are being conducted within the load function. If bulk changes provide significant increase, would be best to refactor this to send all users keys to function rather than one at a time.